webui/tests/tests/StepImplementation.ts

```diff
@@ -149,7 +149,9 @@ export default class StepImplementation {
   @Step('Sign in as <username> with <password>')
   public async signInWithPassword(username: string, password: string) {
     await goto(`${BASE_URL}/login`);
-    await t.clear(t.focus(t.$('#login_username')));
+    const loginSelector = t.$('#login_username');
+    await t.focus(loginSelector);
+    await t.clear(loginSelector);
```

Comment:
can confirm that this section doesn't crash when the login input is not visible:
```
# Authentication
  ## Sign out	 ✔
  ## Sign in as user without password	 ✘
        Failed Step: Sign in as "user-wo-pw" without password
        Specification: specs/02-authentication.spec:12
        Error Message: CustomSelector with query #login_username is not visible
        Stacktrace: 
        Error: CustomSelector with query #login_username is not visible
            at /Users/calebkang/Projects/determined/webui/tests/node_modules/taiko/lib/actions/pageActionChecks.js:189:11
            at waitAndGetActionableElement (/Users/calebkang/Projects/determined/webui/tests/node_modules/taiko/lib/actions/pageActionChecks.js:156:3)
            at focus (/Users/calebkang/Projects/determined/webui/tests/node_modules/taiko/lib/actions/focus.js:12:12)
            at Object.module.exports.focus (/Users/calebkang/Projects/determined/webui/tests/node_modules/taiko/lib/taiko.js:1038:16)
            at Object.module.exports.<computed> [as focus] (/Users/calebkang/Projects/determined/webui/tests/node_modules/taiko/lib/taiko.js:2612:14)
  ## Sign in as user with password	 ✔ ✔
  ## Sign in	 ✔
```